app/src/androidTest/java/org/schabi/newpipe/local/playlist/LocalPlaylistManagerTest.kt

```diff
@@ -0,0 +1,41 @@
+package org.schabi.newpipe.local.playlist
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.schabi.newpipe.TestApp
+import org.schabi.newpipe.database.AppDatabase
+import org.schabi.newpipe.database.stream.model.StreamEntity
+import org.schabi.newpipe.extractor.stream.StreamType
+import javax.inject.Inject
+
+@RunWith(AndroidJUnit4::class)
+class LocalPlaylistManagerTest {
+    @Inject
+    lateinit var appDatabase: AppDatabase
+    @Inject
+    lateinit var localPlaylistManager: LocalPlaylistManager
```

Comment:
With `LocalPLaylistManager` being the class that is tested, i would not inject it. It is supposed to be tightly coupled

Reply:
It uses the testing version of the database.